### PR TITLE
Change concrete playback to use execFile

### LINKED ISSUE #22: Show progress while generating report
Generating the viewer report can be a long process that happens in the background (after the latest changes). This can cause confusion to the user. Similar to kani, there should be some sort of a progress bar or a loading loop that VSCode can show to the user to inform the user that a process is being run in the background.

### PR DESCRIPTION
**Pull request by jaisnan**
### Description of changes: 

Concrete playback's generate test button used the vscode terminal to run the subprocess which has multiple disadvantages. This PR changes that to use execFile, like we do everywhere else and also now displays a status bar icon to show progress while doing that.

### Resolved issues:

Resolves #22 

### Testing:

* How is this change tested?

* Is this a refactor change?

### Checklist
- [x] Each commit message has a non-empty body, explaining why the change was made
- [x] Methods or procedures are documented
- [x] Regression or unit tests are included, or existing tests cover the modified code
- [x] My PR is restricted to a single feature or bugfix

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 and MIT licenses.

### CODE REVIEW

File: src/debugger/debugger.ts [hunk 1]
```diff
@@ -24,9 +31,17 @@ async function getBinaryPath(): Promise<string | undefined> {
 		const globalConfig = GlobalConfig.getInstance();
 		const kaniBinaryPath = globalConfig.getFilePath();
 
-		const playbackCommand: string = `${kaniBinaryPath} playback -Z concrete-playback --only-codegen --message-format=json`;
-		const output = execSync(`cd ${directory} && ${playbackCommand}`);
+		const playbackCommand: string = `cargo kani playback -Z concrete-playback --only-codegen --message-format=json`;
```
Comment (celinval) [inline src/debugger/debugger.ts:34]: I thought the extension invoked `cargo-kani` directly...

Comment (jaisnan) [reply src/debugger/debugger.ts:34]: It still does, I can clarify that with a comment but if you notice line 44, where we call the exec, we invoke cargo-kani directly. The string on this line is used to construct the arguments to be passed.

File: src/model/kaniRunner.ts [hunk 2]
```diff
@@ -258,7 +268,7 @@ function sendErrorToChannel(output: CommandOutput, args: string[]): void {
 
 // Creates a unique name and adds a channel for the harness output to Output Logs
 function sendOutputToChannel(output: CommandOutput, args: string[]): void {
-	const harnessName = args.at(1)!;
+	const harnessName = args.at(args.length - 1)!;
```
Comment (feliperodri) [inline src/model/kaniRunner.ts:271]: Places like this I'd add [invariants](https://www.npmjs.com/package/ts-invariant) (e.g., `args.length != 0`).

**Review by feliperodri — APPROVED** [review]
LGTM.

File: src/model/kaniRunner.ts (hunk 2 above)
Comment (jaisnan) [reply src/model/kaniRunner.ts:271]: Can do! Thanks for pointing it out

**Pull request merged**